CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   (<https://github.com/opencv/cvat/pull/6918>)
 
 ### Fixed
+- Can not input Chinese correctly in text attribute (<https://github.com/opencv/cvat/pull/6916>)
```

Comment:
@yangxiuwu , you don't need to modify the file directly. Please look at the beginning of the file and see the instructions.